content/blog/2021/09/2021-09-22-fortune-500-real-world-results.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "New eBook: Fortune 500 Developers and Engineers Turn to Jenkins for Real-World Results"
+tags:
+- jenkins-is-the-way
+- case-study
+- ebook
+- community
+- announcement
+author: alyssat
+description: >
+  An eBook that focuses on Jenkins user stories in Fortune 500 companies.
+  The book is sponsored by CloudBees.
+opengraph:
+  image: /images/post-images/jenkins-is-the-way/ebook/ebook3_opengraph.png
+links:
+  discourse: true
```

Comment:
was this intentional? or just copy and pasted from last blog? I'm really excited to see comments being enabled, i just want to make sure its intention.